### PR TITLE
[Workspace] Support linked storage accounts

### PR DESCRIPTION
**Pull request by george-moussa**

### CODE REVIEW

File: src/Azure/Azure.Quantum.Client/generated/IJobsOperations.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 // regenerated.
 // </auto-generated>
 
-namespace Microsoft.Azure.Quantum.Client
+namespace Microsoft.Quantum
```
Comment (rmshaffer) [inline src/Azure/Azure.Quantum.Client/generated/IJobsOperations.cs:11]: Was it intentional to change the namespace of the generated code? My feeling is that "Azure" should still be in there somewhere.

Comment (george-moussa) [reply src/Azure/Azure.Quantum.Client/generated/IJobsOperations.cs:11]: i used the swagger we currently have. I believe @anpaz-msft updated it to Microsoft.Quantum

Comment (rmshaffer) [reply src/Azure/Azure.Quantum.Client/generated/IJobsOperations.cs:11]: The namespace can be specified via the `--namespace` argument to `autorest`. I think the reason for previously using `--namespace=Microsoft.Azure.Quantum.Client` was to be consistent with the package name, which is `Microsoft.Azure.Quantum.Client`.

Comment (george-moussa) [reply src/Azure/Azure.Quantum.Client/generated/IJobsOperations.cs]: i synced with Andres and reverted it.

**Pull request merged**